### PR TITLE
Add docker-compose example

### PR DESCRIPTION
**Pull request by as51340**
Add docker-compose deployment which uses custom bridge network.

### CODE REVIEW

File: docker-compose/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Instructions
```
Comment (antejavor) [inline docker-compose/README.md:1]: Can you just elaborate on why this exists in this repository? "To test locally HA..."

File: docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+networks:
```
Comment (antejavor) [inline docker-compose/docker-compose.yml:1]: If we decide to add more compose files, maybe this file should be `ha-docker-compose.yaml`

**Review by antejavor — CHANGES_REQUESTED** [review]

File: docker-compose/README.md (hunk 1 above)
Comment (as51340) [reply docker-compose/README.md:1]: why not? I think users should now that they must change license info

File: docker-compose/docker-compose.yml (hunk 2 above)
Comment (as51340) [reply docker-compose/docker-compose.yml:1]: we can rename it then, at this point it is fine

File: docker-compose/README.md (hunk 1 above)
Comment (antejavor) [reply docker-compose/README.md:1]: Haha, we have a misunderstanding, I was referring to elaborate in README.md why the docker-compose exists and what is the idea behind it :D

**Review by antejavor — APPROVED** [review]

**Pull request merged**